components/common_safe_work_procedures.mdx

```diff
@@ -0,0 +1,24 @@
+Clearpath Robotics recommends the following procedures be followed to reduce the likelihood of harm.
+
+- Maintain a safe distance of at least ten meters from the robot whenever possible.
```

Comment:
Suggest changing to '10 m'
Actual numbers are preferred for technical writing in most cases.

Reply:
Updated